docs/src/main/tut/datatypes/either.md

```diff
@@ -212,132 +212,6 @@ magic("123") match {
 }
 ```
 
-## Either in the small, Either in the large
-Once you start using `Either` for all your error-handling, you may quickly run into an issue where
```

Comment:
We should strive to present certain patterns (for lack of a better word) for using these data types - in this case I wrote it to show how one could use `Either` throughout their code to represent possibly failing computations.